tests/framework/installer/ceph_installer.go

```diff
@@ -127,13 +129,20 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		h.k8shelper.ChangeHostnames()
 	}
 
-	rookOperator := h.Manifests.GetRookOperator(namespace)
+	rookOperator := h.Manifests.GetRookOperator(namespace, h.enableCSI)
 
 	_, err = h.k8shelper.KubectlWithStdin(rookOperator, createFromStdinArgs...)
 	if err != nil {
 		return fmt.Errorf("Failed to create rook-operator pod : %v ", err)
 	}
 
+	if h.rookVersion != Version0_9 {
```

Comment:
With #3196 the integration tests will no longer run v0.9. Assuming we merge that first, does that remove the need for this check?

Reply:
removed the check